.github/workflows/build.yml

```diff
@@ -46,8 +46,8 @@ jobs:
         cache: maven
     - name: Build with Maven for ${{ matrix.os }}
       run: mvn compile
-    - name: Package Drifty CLI for ${{ matrix.os }} with JavaPackager Maven plugin
-      run: mvn javapackager:package@bundling-Drifty-CLI-for-${{ matrix.os }}
+    - name: Package Drifty CLIString for ${{ matrix.os }} with JavaPackager Maven plugin
```

Comment:
@EasyG0ing1 Please don't make any changes in workflow file.

Reply:
This was not intentional. It must have happened accidentally with a global search and replacement within IntelliJ.

Reply:
Ok